Error when running npm install in `kibbeh`

**Describe the bug**
Tried to install dependencies in `kibbeh` but got a Not Found error.

```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@dogehouse%2fkebab - Not found
npm ERR! 404 
npm ERR! 404  '@dogehouse/kebab@^0.1.2' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'kibbeh'
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

**To Reproduce**
Steps to reproduce the behavior:
1. Go to 'kibbeh'
2. Run  'npm install'

**Expected behavior**
Should install dependencies without errors.

**What device are you on?**
OS: Linux
Node version: v14.16.0
npm verson: v6.14.11

**Additional context**
Add any other context about the problem here.

--- a/kibbeh/package.json
+++ b/kibbeh/package.json
@@ -15,7 +15,7 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
-    "@dogehouse/kebab": "^0.1.2",
+    "@dogehouse/client": "^0.1.2",
     "@svgr/webpack": "^5.5.0",
     "date-fns": "^2.19.0",
     "formik": "^2.2.6",

